Tasks/AzureFileCopy/Utility.ps1

```diff
@@ -2,12 +2,32 @@
 
```

Comment:
hold this PR as well, till you publish updated DeploymentUtilities.zip and then need to update the task version

Reply:
Updated DeploymentUtilities.zip url and patch version